src/content/learn/manipulating-the-dom-with-refs.md

```diff
@@ -341,9 +341,9 @@ In this example, `itemsRef` doesn't hold a single DOM node. Instead, it holds a
 
 This lets you read individual DOM nodes from the Map later.
 
-<Canary>
+<NextMajor>
 
-This example shows another approach for managing the Map with a `ref` callback cleanup function.
+Starting in React 19 callback refs can return a cleanup function. When the cleanup function is provided, React will not call the `ref` callback with `null` and call the cleanup function instead:
```

Comment:
Should we mention beta here as well? 
```suggestion
Starting from React 19 beta, callback refs can return a cleanup function. When the cleanup function is provided, React will not call the `ref` callback with `null` and call the cleanup function instead:
```

Reply:
Added the comma, but I think "starting in" works better here, and since this is already versioned to the beta I don't think we need to mention the beta inline.